content/about/introduction/introduction.html

```diff
@@ -14,166 +14,161 @@
-          Further information about platform and browser support is conveyed in the
-          <a href="../../practices/read-me-first/read-me-first-practice.html">Read Me First</a>.
+          APGにはパターンと機能例のライブラリが含まれていますが、包括的なデザインシステムや本番環境に適したコードを提供することは目的に含まれていません。
+          本番環境に適したデザインシステムを開発することは、オーサリングプラクティスタスクフォースの能力を超えるだけでなく、統一されたデザインシステムの要件を満たす例を開発することが、その主な目的である関連する実践の理解を構築することを妨げるでしょう。
+          APGの例、コードスタイルを含む、は、それらが説明するアクセシビリティの概念を学ぶのを容易にするように設計されています。
+          異なるデザインアプローチがアクセシビリティ要件を満たす方法を示すために、例のスタイルと動作は意図的に互いに一貫性がないことがあります。
+          さらに、例は現実的であることが意図されており、現代的なコーディング技術を使用していますが、その目的はウェブエンジニアリングの他の側面、例えば、ローカライゼーションや堅牢なプラットフォーム相互運用性を教えることではありません。
+          プラットフォームおよびブラウザのサポートに関する詳細情報は、<a href="../../practices/read-me-first/read-me-first-practice.html">Read Me First</a>で伝えられています。
         </p>
       </section>
 
       <section id="prerequisite-knowledge">
-        <h2>Important Prerequisite Knowledge</h2>
+        <h2>重要な前提知識</h2>
         <p>
-          The APG Task Force strives to make the APG welcoming to a broad audience by using plain and simple language wherever possible.
-          Consequently, even readers with minimal understanding of web development or accessibility can benefit from a significant portion of the guidance, e.g., the keyboard interaction patterns.
-          At the same time, many topics addressed by the APG are inherently complex and call for technically precise terminology.
-          Additionally, some important aspects of creating accessible experiences fall outside the scope of the APG.
-          This section describes the types of prerequisite knowledge and supporting resources that may enable readers to derive maximum benefit from the APG.
+          APGタスクフォースは、可能な限り平易でシンプルな言語を使用することで、広い視聴者にAPGを歓迎するよう努めています。
+          その結果、ウェブ開発やアクセシビリティの理解が最小限の読者でも、例えば、キーボードインタラクションパターンなど、ガイダンスのかなりの部分から利益を得ることができます。
+          同時に、APGによって対処される多くのトピックは本質的に複雑であり、技術的に正確な用語を必要とします。
+          さらに、アクセシブルな体験を作成するためのいくつかの重要な側面は、APGの範囲外にあります。
+          このセクションでは、APGから最大の利益を得るために読者が有する可能性のある前提知識の種類とサポートリソースを説明します。
         </p>
         <p>
-          Broadly speaking, the APG blends accessibility, UI design, and web engineering.
-          To successfully utilize the APG, it is not necessary for readers to have working knowledge in all three disciplines.
-          Accessibility specialists will find the APG more beneficial with at least a rudimentary understanding of HTML and CSS while designers and engineers will need familiarity with some accessibility fundamentals.
-          All readers will need a firm grasp of some basic ARIA concepts.
+          広く言えば、APGはアクセシビリティ、UIデザイン、およびウェブエンジニアリングを組み合わせています。
+          APGを成功裏に利用するためには、読者がこれら3つの分野すべてにおいて実務知識を持っている必要はありません。
+          アクセシビリティの専門家は、HTMLとCSSの基本的な理解があればAPGからより多くの利益を得るでしょうが、デザイナーやエンジニアはアクセシビリティの基本的な知識が必要になります。
```

Comment:
特に逆接的なニュアンスはないようにおもったので、whileは「一方」としてみました

```suggestion
          アクセシビリティの専門家は、HTMLとCSSの基本的な理解があればAPGからより多くの利益を得るでしょう。一方、デザイナーやエンジニアはアクセシビリティの基本的な知識が必要になります。
```